packages/tldraw/src/lib/tools/selection-logic/updateHoveredShapeId.ts

```diff
@@ -32,7 +32,16 @@ function _updateHoveredShapeId(editor: Editor) {
 }
 
 /** @internal */
-export const updateHoveredShapeId = throttle(
+export const updateHoveredShapeIdThrottled = throttle(
 	_updateHoveredShapeId,
 	process.env.NODE_ENV === 'test' ? 0 : 32
 )
+
+/** @internal */
+export const updateHoveredShapeIdDebounced = debounce(
```

Comment:
throttle vs debounce is one of those things that can take a bit of thinking to really understand the difference between. Are there names we can choose here that make it clearer which option should be used in which scenario? If not, could we add doc comments explaining which should be used in which scenario and why?